person.rb

```diff
@@ -20,3 +26,6 @@ def is_of_age?
   end
   # rubocop:enable Naming/PredicateName
 end
+
+pers = Person.new(age: 10)
```

Comment:
* Kindly ensure that your decorations work by trying out the code snippet in the project requirement, as i got an error when I try to test your decorations 
![image](https://user-images.githubusercontent.com/52497246/219788377-93868e55-3c75-42e3-9e2b-561c53f69300.png)
![image](https://user-images.githubusercontent.com/52497246/219788522-b1a9b616-ee3d-4879-900a-cb1caa08435c.png)